languages/JavaScript/exercises/concept/arrays-basic/.docs/introduction.md

```diff
@@ -0,0 +1,5 @@
+In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or ojects, even mixed types.
```

Comment:
One option could be to also mention that individual items in an array can be accessed by their index.